pynamodb/attributes.py

```diff
@@ -768,7 +768,7 @@ def deserialize(self, values):
             if attr_class is None:
                 continue
             deserialized_value = None
-            if attr_value is not None:
+            if attr_value is not None and attr_class is not None:
```

Comment:
Isn't the `if attr_class is None` above already causing us never to get here if `attr_class` is None?

Should we have a test for this case?